ooc-cli.use

```diff
@@ -0,0 +1,6 @@
+Name: ooc CLI
+Description: CLI helpers
+SourcePath: source/cli
+Requires: ooc-collections
```

Comment:
I think we should very seriously consider removing this clause from all of our `use` files.